### PR TITLE
fix: Add 'Inputs and Outputs' section #2

### PR DESCRIPTION
**Pull request by manekinekko**
I just added code for angularjs, angular and react.
I also fixed a typo.

### CODE REVIEW

**Comment by manekinekko** [comment]
Sorry. I didn't see #7 when I started working on this. Feel free to merge or close this RP ;)

**Comment by feimosi** [comment]
No problem, actually #7 added only Angular snippet, so do you mind merging your changes into it? Preferably unifying the code example across those three frameworks :)

**Comment by manekinekko** [comment]
Let me check that.

**Comment by manekinekko** [comment]
Would you like to keep the `Settings` class example from #7 or keep `CoursesListController`?

**Comment by manekinekko** [comment]
I'll keep the `Settings` class. A fix is on its way.

**Comment by manekinekko** [comment]
done.

File: README.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ const component = {
 
 ### React
 ```jsx
-class CoursesListController {
+class UserPreviewComponent {
```
Comment (feimosi) [inline README.md]: These three should stay untouched (it's different example).

File: README.md [hunk 2]
```diff
@@ -342,6 +363,85 @@ export class UserPreview {
...
+### React
+
+```js
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class UserPreviewComponent extends React.Component {
+  constructor(props) {
+    super(props);
+  }
```
Comment (feimosi) [inline README.md]: No need for the `constructor`.

File: README.md [hunk 3]
```diff
@@ -342,6 +363,85 @@ export class UserPreview {
...
+  editedUser(user: User){
+    console.log('Name of the edited user is', user.name);
+  }
+}
+```
+
+### React
+
+```js
```
Comment (feimosi) [inline README.md]: GitHub has `jsx` syntax too :atom_symbol:

File: README.md [hunk 4]
```diff
@@ -342,6 +363,85 @@ export class UserPreview {
...
+          email: 'foobar@example.com'
+        }
+    };
+  }
+  editedUser(user: User){
+    console.log('Name of the edited user is', user.name);
+  }
+  render() {
+    return (
+      <UserPreviewComponent user={this.state.user} onEdit={(user) => this.editedUser(user)}>
```
Comment (feimosi) [inline README.md]: Can be self-enclosed `<UserPreviewComponent ... />`

**Review by feimosi — CHANGES_REQUESTED** [review]
Great, almost there :slightly_smiling_face:

**Comment by manekinekko** [comment]
fixed.

**Comment by feimosi** [comment]
Great, thanks!

**Review by feimosi — APPROVED** [review]

**Pull request merged**